addons/editor-theme3/theme3.css

```diff
@@ -29,3 +29,31 @@
   fill: var(--editorTheme3-commentText);
   color: var(--editorTheme3-commentText);
 }
+
+.sa-colormode-submenu li:nth-child(1) div,
+.sa-colormode-submenu li:nth-child(2) div {
+  filter: opacity(0.5) grayscale(1);
+}
+.sa-colormode-submenu li:nth-child(1) div img:first-child,
+.sa-colormode-submenu li:nth-child(2) div img:first-child {
+  visibility: hidden;
+}
+.sa-theme3-link div img:first-child {
+  display: none;
```

Comment:
```suggestion
  visibility: hidden;
```
The menu would look better if the icons were aligned.
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/f5d9339a-3c93-4b03-b9a7-0f753164c925)

Reply:
Neutral on this. My concept screenshot (https://github.com/ScratchAddons/ScratchAddons/issues/5905#issuecomment-1559805391) didn't align the "Addon Settings" link because it was a button instead of a menu option.